datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -204,6 +204,8 @@ def __init__(self, *args, **kwargs):
         self.disable_generic_tags = (
             is_affirmative(self.instance.get('disable_generic_tags', False)) if instance else False
         )
+
+        self.global_metrics_filter = self.instance.get('global_metrics_filter') if instance else None
```

Comment:
We should probably at least validate that the regular expression is valid and we won't crash later when checking metrics.

Reply:
Sure, makes sense!